be/src/exec/pipeline/exchange/sink_buffer.h

```diff
@@ -165,6 +165,9 @@ class SinkBuffer {
     int64_t _pending_timestamp = -1;
     mutable std::atomic<int64_t> _last_full_timestamp = -1;
     mutable std::atomic<int64_t> _full_time = 0;
+
+    std::atomic<int64_t> _first_send_time = -1;
```

Comment:
Do we really need to use atomic variable for profile counters? I think for profile counters, using simple variable is OK.

Reply:
Yes, it is, inconsistency is acceptable